HotTips/Tips/EDI_IfElseRefactor.md

```diff
@@ -0,0 +1 @@
+﻿Convert an If-Else loop into a Switch case block  and vice-versa by pressing **Ctrl+.** when your caret is on the if/switch keyword.
```

Comment:
...and then what. Ctrl+. only opens the lightbulb.

How about adding " and selecting "Convert to If-Else or Convert to Switch"?